src/mokka/src/test/java/pl/hycom/mokka/stubbing/responsetemplating/GroovyResponseTransformerTest.java

```diff
@@ -0,0 +1,81 @@
+package pl.hycom.mokka.stubbing.responsetemplating;
+
+import com.github.tomakehurst.wiremock.http.HttpHeaders;
+import com.github.tomakehurst.wiremock.http.Request;
+import com.github.tomakehurst.wiremock.http.RequestMethod;
+import com.github.tomakehurst.wiremock.http.ResponseDefinition;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+/**
+ * @author Bartosz Kuron (bartosz.kuron@hycom.pl)
+ */
+
+public class GroovyResponseTransformerTest {
```

Comment:
Could you please also cover case when `responseDefinition.body` does not contain groovy-script but just regular text?
The expected behavior would be to return original body value.